packages/shared/web/src/header/invest-and-earn/use-invest-earn-links.tsx

```diff
@@ -42,10 +42,22 @@ export const useInvestEarnLinks = () => {
         <FormattedMessage
           defaultMessage="Earn up to {rate} APY"
           values={{
-            rate: formatNumberAsPercent(fCashLend?.totalAPY || 0),
+            rate: fCashLend?.totalAPY ? (
```

Comment:
Can you make the comparison for all of these `fCashLend?.totalAPY === undefined`? If the value is actually zero then we will end up with an endless spinner.

Also these need to be added to the Borrow dropdown as well. I think we can make a little helper function for it to reduce the code.